CHANGELOG.md

```diff
@@ -5,6 +5,154 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project
 tries to adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.6.0](https://github.com/oscar-system/Oscar.jl/releases/tag/v1.6.0) - 2025-11-12
+
+The following gives an overview of the changes compared to the previous release. This list is not
+complete, many more internal or minor changes were made, but we tried to only list those changes
+which we think might affect some users directly.
+
+### Algebraic Geometry
+
+#### New features or extended functionality
+
+- [#5403](https://github.com/oscar-system/Oscar.jl/pull/5403) Add `factoring_standard_basis` command which uses `facstd` from `Singular.jl`
+
+### Combinatorics
+
+#### New features or extended functionality
+
+- [#5477](https://github.com/oscar-system/Oscar.jl/pull/5477) Add `multicombinations` to Oscar
+
+#### Improvements or additions to documentation
+
+- [#5431](https://github.com/oscar-system/Oscar.jl/pull/5431) Improve Combinatorics documentation
+
+### Groups
+
+#### New features or extended functionality
+
+- [#5250](https://github.com/oscar-system/Oscar.jl/pull/5250) Add `fixed_points` for `PermGroup` and `PermGroupElem`
+- [#5270](https://github.com/oscar-system/Oscar.jl/pull/5270) Add an iterator of subspaces of finite vector spaces
+- [#5453](https://github.com/oscar-system/Oscar.jl/pull/5453) Add `character_degrees` for a given group or character table, or for given invariants of a finite abelian group and finite field
+
+### Lie Theory
+
+#### Only changes experimental parts of OSCAR
+
+- [#4570](https://github.com/oscar-system/Oscar.jl/pull/4570) Extend functionality to Demazure modules
```

Comment:
something is wrong here. This PR was already part of 1.5.0, but is listed here again...
ping @aaruni96 @fingolfin